### ISSUE TITLE
Filter FOV visibility by stations in "Filter stations" field

### ISSUE THREAD

**Issue by satmonkey** [tammojan/meteormap#8]
This would improve visibility of FOV's in crowded areas, moreover clicking on station location would show its FOV

**Comment by tammojan** [comment]
Agreed, this would be very nice. I've done this previously hacking in the console. It's a good idea to combine it with the filter box.
![fov-de000c](https://user-images.githubusercontent.com/7974906/147371468-273ab2dd-26b6-405a-bb0e-a5881e456b09.png)
